### DISCUSSION TITLE
중간 점검(06/15) 목표 설정

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#87] [General]
중간 점검 동영상 제출을 위해 목표를 설정해 놓는 것이 좋을 것 같습니다!
(@Hyevvy 혜경님의 아이디어)

**Comment by Hyevvy** [comment]
서버통신전까지 
페이지 구성까지는 다 하기  !! (피그마 모습이 나오게)

**Comment by NamgyungKim** [comment]
[메인페이지] 
 - [ ] text검색기능
 - [ ] 채널검색기능

[card 컴포넌트]
 - [ ] 카드클릭시 상세페이지로 이동하도록 

++ 추가 자잘한 미완료기능

기본적으로 동작해야할 기능들만 완료할 생각입니다~

**Comment by hyunjin0910** [comment]
[로그인 페이지]
- [ ] 레이아웃 사용해서 페이지 만들기
- [ ] 테스트용 계정으로 로그인 성공하기

[회원 가입]
- [ ] 간단하게 페이지 만들고 API 통신되는지 테스트 해보기

**Comment by YuHyun-P** [comment]
[프로필페이지] 더미 데이터로 피그마에 페이지 구성 렌더링
- [x] Cover 컴포넌트 구현
- [ ] EditIcon 컴포넌트 구현
- [ ] TabItem, TabGroup 컴포넌트 구현
- [ ] 팔로워 / 팔로잉 / 온도(변수명 미정..) 컴포넌트 구현
- [ ] Modal 컴포넌트 구현

**Comment by hwoo3303** [comment]
[상세 페이지]
- [ ] 페이지 레이아웃 구성
- [ ] 더미 데이터로 확인
- [ ] 댓글 컴포넌트 구현
- [ ] 댓글 등록 기능 구현
